src/Handler/TagHandler.php

```diff
@@ -73,7 +61,11 @@ public function getTagsHeaderName()
      */
     public function getTagsHeaderValue()
     {
-        return implode(',', array_unique($this->escapeTags($this->tags)));
+        if (empty($this->tags)) {
+            return '';
+        }
+
+        return $this->client->getTagsHeaderValue($this->tags);
```

Comment:
do we need to forward this call to the client? but if we do, i think we should remove the check for no tags and let the client handle this. (currently it should work fine, implode of an empty array returns an empty string)